cmd/trust/sudi.go

```diff
@@ -3,122 +3,106 @@ package main
 import (
 	"crypto/x509"
 	"crypto/x509/pkix"
-	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
 	"time"
 
-	"github.com/apex/log"
 	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/project-machine/trust/pkg/trust"
 	"github.com/urfave/cli"
 )
 
-var genSudiCmd = cli.Command{
-	Name:      "gen-sudi",
+var sudiCmd = cli.Command{
+	Name:      "sudi",
 	Usage:     "Generate and sign sudi cert",
 	UsageText: "CACert, private-key, output-dir, product-uuid [, machine-uuid]",
-	Action:    doGenSudi,
+	Subcommands: []cli.Command{
+		cli.Command{
+			Name:      "list",
+			Action:    doListSudi,
+			Usage:     "list sudi keys",
+			ArgsUsage: "<keyset-name> <project-name>",
+		},
+		cli.Command{
+			Name:      "add",
+			Action:    doGenSudi,
+			Usage:     "add a new sudi key to project",
+			Flags: []cli.Flag{
+				cli.StringFlag {
+					Name: "uuid",
+					Usage: "specify a machine-uuid.  If unspecified, use random.",
+				},
+			},
+			ArgsUsage: "<keyset-name> <project-name> <serial-number>",
+		},
+	},
 }
 
+// ~/.local/share/machine/trust/keys/
+//      keyset1/manifest/project-name/{uuid,privkey.pem,sudi.pem}
```

Comment:
should this be "manifest.pem" or "cert.pem" instead of sudi.pem?

Reply:
yeah it should be cert.pem.  It was however a note to self :)  I should probably just remove it.